{
  try {
    resolvedConfiguration.rethrowFailure();
  }
 catch (  Throwable e) {
    throw wrapException(e,resolveContext);
  }
}
